docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -1335,6 +1335,18 @@ Some hash types allow providing a rounds parameter::
     {{ 'secretpassword' | password_hash('sha256', 'mysecretsalt', rounds=10000) }}
     # => "$5$rounds=10000$mysecretsalt$Tkm80llAxD4YHll6AgNIztKn0vzAACsuuEfYeGP7tm7"
 
+Hash type 'blowfish' (BCrypt) provides facility to specify the version of the BCrypt algorithm::
```

Comment:
```suggestion
Hash type 'blowfish' (BCrypt) provides the facility to specify the version of the BCrypt algorithm::
```

Reply:
please change this to code-blocks. Translation doesn't work well with the :: option.